L1Trigger/L1THGCal/src/concentrator/HGCalConcentratorSuperTriggerCellImpl.cc

```diff
@@ -13,8 +14,20 @@ HGCalConcentratorSuperTriggerCellImpl::getSuperTriggerCellId(int detid) const {
     return TC_id.cell(); //scintillator
   } else {
     int TC_wafer = TC_id.wafer();
-    int TC_12th = ( TC_id.cell() & kSplit_ );
-    return TC_wafer<<kWafer_offset_ | TC_12th;
+    int split_12th = 0x3a;
```

Comment:
Similarly to `kWafer_offset_` and `kSplit_`, these should be static const class members.